docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 
 # Basic Components
 
-Nodes work with a data records. They can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
+Nodes work with data records. They can produce, fetch, send and collect data or organize the flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
 
-Most of the nodes, with source and sink nodes being notable exceptions, have both input and at least one output flow.
+Most of the nodes have both input and at least one output flow. Source and sink nodes are an exception.
 
 Sinks and filters can be disabled by selecting `Disable` checkbox.
 
 &nbsp;
 ## Variable
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable holds its content and  its value is assigned to a newly declared `myFirstVariable` variable.
+A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the event has been read from a Kafka topic, the `#input` variable holds event contents. Event value is assigned to a newly declared `myFirstVariable` variable.
```

Comment:
Variables are not easy to. We ended up with the following statement on variables [here](https://nussknacker.io/documentation/docs/1.12/scenarios_authoring/Intro/#variables): _Variables are attributes of the data record, they do not exist by themselves._

In this context stating _event's value_ will be confusing. Event does not have value, it has attributes in the form of variables. For me correct formulations would be _content of #input variable is assigned to myFirstVariable_.

Finally, this fragment of docs was written when we did not have R-R processing mode, so term "event" is used. May be it could be replaced with more neutral "data record" term, or even better, we could run away from mentioning Kafka here.

Reply:
I agree, but I would like to underline the fact illustrated in the graphic that first the record value is assigned to input and then to another variable. I have changed the wording to underline that and omit kafka.